packages/avatars/src/types/index.ts

```diff
@@ -24,6 +24,14 @@ export interface IAvatarProps extends HTMLAttributes<HTMLElement> {
   size?: typeof SIZE[number];
   /** Applies status styling */
   status?: typeof STATUS[number];
-  /** Sets the badge text and applies active styling */
-  badge?: string | number;
+  /** Sets the badge and applies active styling if it is a string or number */
+  badge?: string | number | boolean | ReactNode;
```

Comment:
`ReactNode` includes `string | number | boolean` in its type definition. Can this simplify to `badge?: ReactNode;`

<img width="768" alt="Screen Shot 2022-05-31 at 12 03 24 PM" src="https://user-images.githubusercontent.com/1975639/171265326-563ecd15-ea55-451f-bb35-9e383d8690b7.png">
<img width="392" alt="Screen Shot 2022-05-31 at 12 03 32 PM" src="https://user-images.githubusercontent.com/1975639/171265331-5f122a08-af88-469f-8630-d9b37a2386b1.png">